javascript/operators/new_target.json

```diff
@@ -0,0 +1,61 @@
+{
+  "javascript": {
+    "operators": {
+      "new_target": {
```

Comment:
Should this be at this level or be javascript.operators.new.target ?

Reply:
No, I think new.target is its own feature, but the naming with the dot is just weird.